modules/configuring-egress-proxy.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/configure-egress-proxy-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-egress-proxy_{context}"]
+= Configuring the cluster wide egress proxy on the AWS Load Balancer Operator
+
+After configuring the cluster wide egress proxy on the AWS Load Balancer Operator in the {product-title} cluster, Operator Lifecycle Manager (OLM) automatically updates all the deployments of the Operators with the environment variables such as `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY`.
+
+You must trust a custom Certificate Authority (CA) and create the OLM subscription of the AWS Load Balancer Operator.
```

Comment:
```suggestion
= Configuring the AWS Load Balancer Operator to trust the cluster wide egress proxy's Certificate Authority (CA)
```

The details about the env variables are common to all the proxy users, they are well described in the egress proxy sections of the OCP docs. We can save some text here.

Reply:
Addressed